h5g_group.go

```diff
@@ -145,6 +173,17 @@ func (g *CommonFG) ObjectNameByIndex(idx uint) (string, error) {
 	return C.GoString(&name[0]), nil
 }
 
+// ObjectTypeByIndex returns the type of the object at idx.
+func (g *CommonFG) ObjectTypeByIndex(idx uint) (GType, error) {
+	cidx := C.hsize_t(idx)
+	ctyp := C.H5Gget_objtype_by_idx(g.id, cidx)
+	if ctyp < 0 {
```

Comment:
Does `GType(0)` have a name?

Reply:
yes, that's `H5G_GROUP`.
I'll update that.